libs/labelbox/tests/data/serialization/ndjson/test_video.py

```diff
@@ -491,16 +499,13 @@ def test_video_classification_frameline():
 
     label = Label(data=VideoData(global_key="sample-video-4.mp4",),
                   annotations=bbox_annotation)
-
     serialized = NDJsonConverter.serialize([label])
     res = [x for x in serialized]
-    for annotations in res:
-        annotations.pop("uuid")
     assert res == expected
 
     deserialized = NDJsonConverter.deserialize(res)
     res = next(deserialized)
     annotations = res.annotations
-    for annotation in annotations:
+    for i, annotation in enumerate(annotations):
         annotation.extra.pop("uuid")
-    assert annotations == label.annotations
+        assert annotation.value == label.annotations[i].value
```

Comment:
equalities of base models do not behave the same in V2 I spent a long time figuring that out (comparing each field even printing the model full contents to find the difference) I had to change this up to either dump the model and compare or compare fields. The models dump to the correct format which is what is tested above this is just an addition to tests